docker/pravega/Dockerfile

```diff
@@ -14,12 +14,16 @@ RUN apk add --update \
     nfs-utils \
     libc6-compat \
     python \
+    jq \
+    curl \
   && rm -rf /var/cache/apk/*
 
 EXPOSE 9090 9091 10000 12345
+
 WORKDIR /opt/pravega
+
 COPY pravega/ /opt/pravega/
-COPY entrypoint.sh /opt/pravega/scripts/
-COPY wait_for /opt/pravega/scripts/
+COPY scripts/ /opt/pravega/scripts/
+RUN chmod +x -R /opt/pravega/scripts/
```

Comment:
Just wanted to highlight that Docker daemon postversion `17.09.0-ce`  supports `--chown flag to ADD/COPY commands in Dockerfile ` ref: https://docs.docker.com/engine/release-notes/

Reply:
@shrids thanks for your suggestion, I was not aware of that feature. Hoever, the flag that you are mentioning is `chown` (change ownership), but what I'm doing is making those scripts executable `chmod +x`, which are two different things. Looks like `chmod` is not supported as a flag of `ADD/COPY` commands.